js-envs/src/main/scala/org/scalajs/jsenv/nodejs/JSDOMNodeJSEnv.scala

```diff
@@ -88,7 +89,14 @@ class JSDOMNodeJSEnv(
            |})();
            |""".stripMargin
       }
-      Seq(new MemVirtualJSFile("codeWithJSDOMContext.js").withContent(jsDOMCode))
+      val codeFile = new File(jsdomDirectory, "codeWithJSDOMContext.js")
```

Comment:
For reference here is an extract of the above linked page (emphasis is *not* mine):

> **You are highly encouraged to place your dependencies locally in `node_modules` folders.**